README.md

```diff
@@ -19,10 +19,11 @@ This Action provides Amazon Web Services S3 backend (and compatible software) fo
     aws-s3-bucket: ${{ secrets.AWS_S3_BUCKET_NAME }}
     aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
     aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-    aws-region: us-east-1                      # Optional
-    aws-endpoint: https://example.com          # Optional
-    aws-s3-bucket-endpoint: false              # Optional
-    aws-s3-force-path-style: true              # Optional
+    aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }} # Optional
```

Comment:
Please write `How to generate token` e.g. generated token from `aws-actions/configure-aws-credentials`

Reply:
Done! Thanks for your work on the S3 backed version of the cache action.